scalatest/src/main/scala/org/scalatest/tools/ScalaTestAntTask.scala

```diff
@@ -1017,8 +1017,8 @@ class ScalaTestAntTask extends Task {
   }
   
   private[tools] class TestElement {
-    private var name: String = null
-    private var substring: String = null
+    private var name: String | Null = null
+    private var substring: String | Null = null
```

Comment:
**Nullable variable**
Variables that really can be null and should therefore have a `| Null` type.